test/parallel/test-buffer-slow.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const buffer = require('buffer');
+const Buffer = buffer.Buffer;
+const SlowBuffer = buffer.SlowBuffer;
+
+const ones = [1, 1, 1, 1];
+
+let sb = new SlowBuffer(4);
+assert(sb instanceof Buffer);
+assert.strictEqual(sb.length, 4);
+sb.fill(1);
+assert.deepEqual(sb, ones);
+
+// should work without new
+sb = SlowBuffer(4);
+assert.strictEqual(sb.length, 4);
+sb.fill(1);
+assert.deepEqual(sb, ones);
+
+// should create zero-length buffer if parameter is not a number
+sb = SlowBuffer();
+assert.strictEqual(sb.length, 0);
+sb = SlowBuffer(NaN);
+assert.strictEqual(sb.length, 0);
+sb = SlowBuffer({});
+assert.strictEqual(sb.length, 0);
+sb = SlowBuffer('string');
+assert.strictEqual(sb.length, 0);
+
+// should throw with invalid length
+assert.throws(function() {
+  new SlowBuffer(Infinity);
+}, RangeError);
+assert.throws(function() {
+  new SlowBuffer(-1);
+}, RangeError);
+
```

Comment:
To make sure it doesn't throw? Sounds like an appropriate test.

On that note might be useful to test `require('buffer').kMaxLength` (upper bound) to make sure it succeeds, and `kMaxLength + 1` to make sure it throws. This is useful because that value depends on architecture.